packages/db-postgres/src/types.ts

```diff
@@ -4,13 +4,26 @@ import type {
   MigrateDownArgs,
   MigrateUpArgs,
   PostgresDB,
+  PostgresSchemaHook,
 } from '@payloadcms/drizzle/postgres'
 import type { DrizzleAdapter } from '@payloadcms/drizzle/types'
 import type { DrizzleConfig } from 'drizzle-orm'
 import type { PgSchema, PgTableFn, PgTransactionConfig } from 'drizzle-orm/pg-core'
 import type { Pool, PoolConfig } from 'pg'
 
 export type Args = {
+  /**
+   * Transform the schema after it's built.
+   * You can use it to customize the schema with features that aren't supported by Payload.
+   * Examples may include: composite indices, generated columns, vectors
```

Comment:
Assuming it is possible to add composite indices, we need to document how that looks in the /docs.
Is that something you can test and try to write up?